test/integration/replicasets/conftest.py

```diff
@@ -239,7 +239,7 @@ def project_with_one_joined_instance(cartridge_cmd, built_project, start_stop_cl
         instances_list=[instance],
     )
 
-    p.start()
+    p.start(stateboard=True)
```

Comment:
Why have you add `stateboard=True` flag everywhere?

It seems that `p.start()` function can accept `(..., stateboard=None)` value and then pass `--stateboard` or `--stateboard=false` if non-none value is specified. `stateboard=False` can be used to overwrite `stateboard: true` value from default `.cartridge.yml`.